dashboard/app/controllers/lti/v1/deep_linking_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Lti
+  module V1
+    class DeepLinkingController < ApplicationController
+      before_action :authenticate_user!
+      before_action :validate_deep_linking_enabled
+
+      # GET /lti/v1/deep_linking
+      def index
+        @deep_linking_settings = params[:deep_linking_settings]
+      end
+
+      private def validate_deep_linking_enabled
+        unless DCDO.get('schoology_deep_linking_enabled', false)
+          head :not_implemented and return
+        end
```

Comment:
We do not really need to `return` here
```suggestion
        head :not_implemented unless DCDO.get('schoology_deep_linking_enabled', false)
```